eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForOpenChannelStateSpec.scala

```diff
@@ -88,6 +92,15 @@ class WaitForOpenChannelStateSpec extends TestkitBaseClass with StateTestsHelper
     awaitCond(bob.stateName == CLOSED)
   }
 
+  test("recv OpenChannel (fundingSatoshis > max-funding-satoshis)", Tag("wumbo")) { f =>
+    import f._
+    val open = alice2bob.expectMsgType[OpenChannel]
+    val highFundingSat = Btc(2).toSatoshi // bob has max-funding-satoshis = 1BTC
```

Comment:
Maybe more future-proof:
```suggestion
    val highFundingSat = Bob.nodeParams.maxFundingSatoshis + Btc(1)
```